packages/preview/amlos/0.2.1/typst.toml

```diff
@@ -2,7 +2,7 @@
 name = "amlos"
 version = "0.2.1"
 entrypoint = "lib.typ"
-authors = ["uwni <https://github.com/uwni>"]
+authors = ["枚鴉 <@uwni>"]
```

Comment:
For info, for reproducibility reasons, we will disallow any edits to previously published packages, even if we used to tolerate metadata changes. It's fine for now because we didn't communicate clearly about it yet, but I let you know for next time.